test/unit/spec/LoginRouter_spec.js

```diff
@@ -1045,7 +1047,7 @@ function (Okta, Q, Logger, Errors, BrowserFeatures, WidgetUtil, Bundles, config,
           return Expect.waitForSpyCall(spy)
             .then(function () {
               expect(spy.calls.count()).toBe(1);
-              expectAuthorizeUrl(spy.calls.argsFor(0)[0], options);
+              options && expectAuthorizeUrl(spy.calls.argsFor(0)[0], options);
```

Comment:
curious what's been called when `options` is null given `spy.calls` is always 1.

Reply:
This was left from an earlier revision where I wanted to wait for the call but not verify the parameters. I can remove it

Reply:
as long as it work, let's remove it to avoid confusing.